packages/firestore/package.json

```diff
@@ -97,8 +97,11 @@
     "@firebase/auth": "0.20.4",
     "@rollup/plugin-alias": "3.1.9",
     "@rollup/plugin-json": "4.1.0",
+    "@types/chai": "^4.3.1",
```

Comment:
The repo's main package.json already has `@types/chai` and `chai` so I don't think it's necessary to add them here.

Reply:
Removed.